src/classes/QueryBuilder.cls

```diff
@@ -2,41 +2,54 @@
 * This file is part of the Nebula Framework project, released under the MIT License.             *
 * See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
 *************************************************************************************************/
-public without sharing class QueryBuilder extends NebulaCore implements IQueryBuilder {
+public without sharing virtual class QueryBuilder extends NebulaCore implements IQueryBuilder {
```

Comment:
ah yes, I remember now - I had an extension for returning multiple objects with SOSL in the first implementation of our "make email address unique" story.  We ended up going with two SOQL calls from Lead / Contact repo's instead.  Apologies again, backing this out